src/libutil/fmt.hh

```diff
@@ -136,4 +139,11 @@ inline hintformat hintfmt(const std::string & fs, const Args & ... args)
     return f;
 }
 
+inline hintformat hintfmt(std::string fs)
+{
+    // we won't be receiving any args in this case, so escape all percents.
+    boost::replace_all(fs, "%", "%%");
+    hintformat f(fs);
+    return f;
+}
```

Comment:
Nit, but I think I rather do
```suggestion
inline hintformat hintfmt(std::string plain_string)
{
    // we won't be receiving any args in this case, so just print the original string
    return hintformat { "%s", plain_string };
}
```
which has the slight advantage of not relying on the format string format, even though that format is unlikely to change.

Reply:
nice!  I like that better too.